src/OrchardCore/OrchardCore.DisplayManagement/SiteServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,13 @@
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Settings;
+
+namespace OrchardCore.DisplayManagement.Handlers;
+
+public static class SiteServiceCollectionExtensions
+{
+    public static IServiceCollection AddSiteDisplayDriver<TDriver>(this IServiceCollection services)
```

Comment:
Let's add another that might accept multiple site display drivers too

Reply:
No need. you can just chain more than one if needed.

Reply:
No problem I just uses the chain method in two places, let's accept my changes then merge